src/core/lib/surface/init_internally.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GRPC_SRC_CORE_LIB_SURFACE_INIT_INTERNALLY_H
+#define GRPC_SRC_CORE_LIB_SURFACE_INIT_INTERNALLY_H
+
+namespace grpc_core {
+
+extern void (*InitInternally)();
```

Comment:
Just for curiosity, this has to conform with C89?

Reply:
no... it's internal... just the top level include/grpc tree needs to

Reply:
or does it have to be C?